yard.gemspec

```diff
@@ -21,4 +21,7 @@ Gem::Specification.new do |s|
   s.executables   = ['yard', 'yardoc', 'yri']
   s.license = 'MIT' if s.respond_to?(:license=)
   s.metadata['yard.run'] = 'yri'
+
+  # FIXME: The following dependency can be removed when dropping legacy Ruby support.
+  s.add_runtime_dependency('e2mmap')
```

Comment:
We can probably get rid of this dep with a small change in slex since it's only used in a few lines of easily replaceable code.